<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(156, 211, 255)
</string>
  </property>
  <widget class="QLabel" name="ea">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)</string>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)
</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="SignUp">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 26pt &quot;Georgia&quot;;
</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subutton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(184, 210, 146)</string>
   </property>
   <property name="text">
    <string>Sign up!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="easu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordsu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>300</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpassword">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)
</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordsu_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
  </widget>
  <widget class="QLabel" name="ea_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)</string>
   </property>
   <property name="text">
    <string>First name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastnamelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)</string>
   </property>
   <property name="text">
    <string>Last name</string>
   </property>
  </widget>
  <widget class="QLabel" name="DOBlabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(84, 116, 197)
</string>
   </property>
   <property name="text">
    <string>Date of birth (MM/DD/YYYY)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DOB">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastname">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="firstname">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:
rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ihaveacct_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>480</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(17, 85, 204)</string>
   </property>
   <property name="text">
    <string>I have an account</string>
   </property>
  </widget>
  <widget class="QLabel" name="errormessage">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff000f;&quot;&gt;An account with that email already exists!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="errormessage2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff000f;&quot;&gt;Password needs to be longer than 6 characters.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
